### PR TITLE
fix: endless loading

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

File: src/app/shared/services/claims/claims.service.ts [hunk 1]
```diff
@@ -47,14 +52,28 @@ export class ClaimsService {
...
+    ).pipe(
+      catchError((err) => {
+        console.error(err);
+        return of(err);
+      }),
+      finalize(() => {
+        if (showLoader) {
+          this.loadingService.hide();
+        }
+      })
```
Comment (jrhender) [inline src/app/shared/services/claims/claims.service.ts:64]: Is the key change that `finalize` with the `loadingService.hide()` was missing? Or more that the `catchError` was missing?

Comment (dawidgil) [reply src/app/shared/services/claims/claims.service.ts:64]: No, i did not hide it. So the key is `finalize`.

Comment (dawidgil) [reply src/app/shared/services/claims/claims.service.ts:64]: I though catchError is needed, but it looks like it's not.

**Review by jrhender — APPROVED** [review]

File: src/app/shared/services/claims/claims.service.ts (hunk 1 above)
Comment (jrhender) [reply src/app/shared/services/claims/claims.service.ts:64]: Ok, thanks!

**Pull request merged**